audformat/core/utils.py

```diff
@@ -353,6 +354,36 @@ def hash(
     return str(pd.util.hash_pandas_object(obj).sum())
 
 
+def index_has_overlap(
+        obj: typing.Union[pd.Index, pd.DataFrame, pd.Series],
+) -> bool:
+    r"""Check if one or more segments in the index overlap.
+
+    Args:
+        obj: object conform to
+            :ref:`table specifications <data-tables:Tables>`
+
+    Returns:
+        ``True`` if overlap is detected, otherwise ``False``
+
+    """
```

Comment:
I think we should add a docstring example.

Reply:
Added, and description above updated.

Reply:
Cool, thanks. I have a slight suggestion though:

```
    Example:
        >>> index = filewise_index(['f1', 'f2'])
        >>> index_has_overlap(index)
        False
        >>> index = segmented_index(
        ...        ['f1', 'f2'],
        ...        [0, 1],
        ...        [2, 3],
        ...    )
        >>> index_has_overlap(index)
        False
        >>> index = segmented_index(
        ...        ['f1'] * 2,
        ...        [0, 1],
        ...        [2, 3],
        ...    )
        >>> index_has_overlap(index)
        True
```

Why? We now start with the simplest example and move on to advanced ones.

Reply:
Make sense, applied.

Reply:
Thanks, could you please update the description then we are good to go I think.

Reply:
Please also update the title to `Add utils.index.has_overlap()` (no need to mention that we add tests).

Reply:
> Thanks, could you please update the description then we are good to go I think.

Already did

> Please also update the title to Add utils.index_has_overlap() (no need to mention that we add tests).

Done now

Reply:
> Already did

It still shows the old example, but ok, not so important.

Reply:
Weird, are you talking about the PR description? Because it's updated for me 

![image](https://user-images.githubusercontent.com/38329838/165305820-3a4bd674-e9a0-4f0f-b829-f96a9c44c074.png)

Reply:
Ok, now for me, too. Maybe it was just a caching issue. Looks great now!